### PR TITLE
Run autorest-fixer to explicitly set modelAsString=false for single value enums

### PR DESCRIPTION
**Pull request by timotheeguerin**
As we talked during the meeting this runs the autorest-fixer tool to explicitly mark all single value enums with modelAsString=false to be able to use the previous behavior.

There is unfortunatelly a tiny bit of noise in the changes as JSON.parse will reorder some keys(specially number) and doesn't seem to be a good way to prevent it.

### CODE REVIEW

File: specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2020-07-01/appplatform.json [hunk 1]
```diff
@@ -87,14 +87,14 @@
           }
         ],
         "responses": {
-          "201": {
-            "description": "Created. The response describes the new Service and contains a Location header to query the\r\noperation result.",
+          "200": {
```
Comment (heaths) [inline specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2020-07-01/appplatform.json:90]: This seems to have nothing to do with setting `modelAsString=false` and is a breaking change.

File: specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2018-02-14-preview/keyvault.json [hunk 2]
```diff
@@ -51,13 +51,13 @@
           }
         ],
         "responses": {
-          "201": {
+          "200": {
```
Comment (heaths) [inline specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2018-02-14-preview/keyvault.json:54]: This seems to have nothing to do with setting `modelAsString=false` and is a breaking change.

**Review by heaths — CHANGES_REQUESTED** [review]
There seems to be some other change here that's changing 201s to 200s which is a breaking change.

File: specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2020-07-01/appplatform.json (hunk 1 above)
Comment (timotheeguerin) [reply specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2020-07-01/appplatform.json:90]: it is just reordering the keys, it is still the same responses. This is due to JSON.parse somehow only deciding to sort and reorder number like keys.

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
Key Vault changes LGTM

**Comment by tjprescott** [comment]
@timotheeguerin are all the breaking change warnings and so forth expected?

**Comment by JackTn** [comment]
Hi, @timotheeguerin. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Comment by JackTn** [comment]
Hi, @timotheeguerin. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Comment by JackTn** [comment]
Hi, @timotheeguerin. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Pull request closed without merging**